### ISSUE TITLE
Model does not converge on CD2014 dataset

### ISSUE THREAD

**Issue by PhiAbs** [gmayday1997/SceneChangeDet#18]
I am currently trying to train the model on the CD2014 dataset (deeplabv2 as a backbone). 
I tried two different methods for training:

- Method 1: I used the predefined parameter for learning, except for the learning rate which I set to 1e-10 (because in an earlier test, I had the impression that 1e-7 was too large). I also adjusted the code such that I can train with batch_size > 1. 
- Method 2: basically the same as for method 1, but I applied gradient clipping with max_norm=5. 

The problem is that the model does not converge for either training method. Did anybody also have that issue? How did you solve it? What parameters did you use for training? How long did you have to train?

**Comment by Syzcannot** [comment]
what is your Pytorch version？
